tests/test_api.py

```diff
@@ -113,3 +126,72 @@ def test_continue_trace(sentry_init):
         assert propagation_context["dynamic_sampling_context"] == {
             "trace_id": "566e3688a61d4bc888951642d6f14a19"
         }
+
+
+@pytest.mark.forked
+def test_is_initialized():
+    assert not is_initialized()
+
+    scope = Scope.get_global_scope()
+    scope.set_client(Client())
+    assert is_initialized()
+
+
+@pytest.mark.forked
+def test_get_client():
+    client = get_client()
+    assert client is not None
+    assert client.__class__ == NoopClient
+    assert not client.is_active()
+
+
+@pytest.mark.forked
+def test_get_current_scope():
+    scope = get_current_scope()
+    assert scope is not None
+    assert scope.__class__ == Scope
+    assert scope._type == "current"
+
+
+@pytest.mark.forked
+def test_get_isolation_scope():
+    scope = get_isolation_scope()
+    assert scope is not None
+    assert scope.__class__ == Scope
+    assert scope._type == "isolation"
+
+
+@pytest.mark.forked
+def test_get_global_scope():
+    scope = get_global_scope()
+    assert scope is not None
+    assert scope.__class__ == Scope
+    assert scope._type == "global"
+
+
+@pytest.mark.forked
+def test_set_current_scope():
+    scope = Scope(ty="test_something")
```

Comment:
having a scope of arbitrary type is not we want anybody to do so this test makes no sense to me. We want to have 3 types of scopes - global/isolated/current (and the merged thing you defined?) and nothing else. Maybe `_type` should be an enum to make it more clear.